db/migrations/development/02_create_cookbooks_table.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('cookbooks', t => {
+        t.increments('id')
+        t.integer('user_id')
+        t.foreign('user_id').references('id').inTable('users')
+        t.string('cookbook_name')
+        t.string('category')
```

Comment:
What is this column purpose?

Reply:
Lu put in the field when creating a cookbook. It makes sense, and it can be thought of as a descriptor. Ex, when making a cookbook, I could say it is a "Mexican" or "Chinese" or "Italian" category, etc. It doesn't have to be like recipes (where you want to filter recipes by their category: drinks, salads, appetizers, etc)